.github/workflows/build.yml

```
@@ -0,0 +1,139 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+  
+  workflow_run:
+    workflows: ["PR handler"]
+    types:
+      - completed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build examples
+
+    if: >
+      ${{ (github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success') ||
+      github.event_name == 'push' }}
+
+    outputs:
+      pr_number: ${{ steps.pr_number.outputs.PR_NUMBER }}
+
+    steps:
+      - name: 'Download artifact'
+        if: ${{ github.event_name == 'workflow_run' }}
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            var fs = require('fs');
+            var artifacts = await github.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: ${{ github.event.workflow_run.id }},
+            });
+            var matchArtifact = artifacts.data.artifacts.filter((artifact) => {
+              return artifact.name == "pr"
+            })[0];
+            var download = await github.actions.downloadArtifact({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               artifact_id: matchArtifact.id,
+               archive_format: 'zip',
+            });
+            fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data));
+      - run: unzip pr.zip
+        if: ${{ github.event_name == 'workflow_run' }}
```

```suggestion
      - name: Unzip artifact
        if: ${{ github.event_name == 'workflow_run' }}
        run: unzip pr.zip
```
To follow a convention